l10n/en/firefox/browsers/quantum.ftl

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/quantum/
+
+the-latest-firefox = The latest { -brand-name-firefox } engine: { -brand-name-firefox } Quantum
+firefox-quantum-was = { -brand-name-firefox } Quantum was a revolution in { -brand-name-firefox } development. In 2017 we created a new, lightning fast browser that constantly improves. { -brand-name-firefox } Quantum is the { -brand-name-firefox-browser }.
+
+# https://blog.mozilla.org/blog/2017/11/14/introducing-firefox-quantum/
+learn-more-about = Learn more about Quantum
+
+privacy-first = Privacy first
+firefox-doesnt-spy = { -brand-name-firefox } doesn’t spy on you online. We stop many known third-party tracking cookies and give you full control.
+super-fast = Super Fast
+get-speed-and = Get speed and security. { -brand-name-firefox } is fast because we don’t track your moves.
+always-evolving = Always evolving
+
+# attrs = https://www.mozilla.org/en-US/firefox/
```

Comment:
Suggest using the standard convention for annotating variables? e.g. https://github.com/mozilla/bedrock/blob/master/l10n/en/download_button.ftl#L16-L17